spec/index.html

```diff
@@ -660,10 +660,10 @@ <h2>RDF Entailment Regime</h2>
               <th>Query Answers</th>
               <td>
                 <p>Let G be the queried RDF graph, BGP be a basic graph pattern, V(BGP) the set of variables in BGP, B(BGP) the set of blank nodes in BGP, SG the
-                  <a data-cite="SPARQL12-QUERY#BGPsparqlBNodes">scoping graph</a> for G and BGP, sk(SG) a <a data-cite="RDF12-SEMANTICS#glossSkolemization">Skolemization</a> of SG with
+                  <a data-cite="SPARQL12-QUERY#BGPsparqlBNodes">scoping graph</a> for G and BGP, sk(SG) a <a data-cite="RDF12-SEMANTICS#skolemization">Skolemization</a> of SG with
                 respect to a vocabulary disjoint from the vocabulary of SG and BGP. Applying sk to a term t, written sk(t), yields sk(t) if sk is defined for t and t otherwise; applying sk to a BGP,
                 written sk(BGP), replaces each blank node b in BGP for which sk is defined with sk(b). The set rdfV contains the URI references of the
-                <a data-cite="RDF12-SEMANTICS#RDFINTERP">RDF vocabulary</a> and rdfV-Minus is the set of URI references in rdfV minus URI references of the form <code>rdf:_n</code>
+                <a data-cite="RDF12-SEMANTICS#RDF vocabulary">RDF vocabulary</a> and rdfV-Minus is the set of URI references in rdfV minus URI references of the form <code>rdf:_n</code>
```

Comment:
`RDF12-SEMANTICS#RDF vocabulary` is an invalid URI... I looked at [RDF12-SEMANTICS](https://w3c.github.io/rdf-semantics/spec/), but I was not able to figure out what this fragment ID should be.

Reply:
@dfn-rdf-vocabulary, but in a PR, fixed (soon)